server/tests/integration/customers.test.ts

```diff
@@ -14,15 +14,19 @@
  * limitations under the License.
  */
 
-import supertest from 'supertest';
+import request from 'supertest';
 
 import app from '../../src';
+import customerFixtures from '../fixtures/customers';
 
 describe('Customers endpoints', () => {
   test('it should fetch a single customer', async () => {
-    // TODO: Do a proper test. This is just a sample test to check that it works.
-    const customerId = 1;
-    const res = await supertest(app).get(`/customers/${customerId}`);
-    expect(res).toBeTruthy();
+    const expectedCustomerData = customerFixtures.data?.[0];
+    const customerId = customerFixtures.ids?.[0];
```

Comment:
Suggestion. Place one blank line in between the Arrange, Act and Assert sections in the unit test.
https://medium.com/@pjbgf/title-testing-code-ocd-and-the-aaa-pattern-df453975ab80

In this example, the `await request(app).get(`/customers/${customerId}`);` is the "Act" which can be in its own section (i.e., add one blank line on top and one blank line below).